README.md

```diff
@@ -2,55 +2,16 @@
 
 [![CI](https://github.com/maplibre/martin/workflows/CI/badge.svg)](https://github.com/maplibre/martin/actions)
 ![Security audit](https://github.com/maplibre/martin/workflows/Security%20audit/badge.svg)
-
+[![Slack chat](https://img.shields.io/badge/Chat-on%20Slack-brightgreen)](https://slack.openstreetmap.us/)  
```

Comment:
thanks!  Want to make it first, or else we have two off-green items next to each other.

Reply:
Make it first now